config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
   
   get 'server', to: 'server#index'
+  get 'notifications', to: 'notifications#notifications'
```

Comment:
You do this DHH's way and do...

```ruby
resources :notifications, only: [:index]
```

Then change your action to `index`, instead of `notifications`